packages/api/src/chats/chat-socket.gateway.ts

```diff
@@ -158,7 +158,7 @@ export class ChatSocketGateway {
       if (!curr.content) {
         return 0;
       }
-      return (acc += encode(curr.content).length);
```

Comment:
what was the issue here?

Reply:
`content` is no longer just a string
https://api.js.langchain.com/types/langchain_core_messages.MessageContent.html